engineio/async/threading.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+
+import threading
+
+try:
+    import queue
+except ImportError:  # pragma: no cover
+    import Queue as queue
+Queue = queue.Queue
+QueueEmpty = queue.Empty
+has_websocket = False
+
+
+def thread(target):
+    return threading.Thread(target=target)
+
+
+def wrap_websocket(app):
+    raise RuntimeError("Websocket support is not available")
```

Comment:
I went with additional boolean flag because `Server._upgrades` wants to know whenever there is WS support available or not. I could've also made it that `wrap_websocket(None)` would be a convention to check if the support is available without actually wrapping anything, but boolean variable looked simpler.